### ISSUE TITLE
Does not work with firefox developer tools

### ISSUE THREAD

**Issue by claudenobs** [benmmurphy/jsdeminifier_xpi#12]
Strangely it doesn't work with firefox's own developer tools. i really prefer working with those since using firebug makes firefox extremely sluggish...

**Comment by benmmurphy** [comment]
yeah. this is quite annoying :( it might be because firefox developer tools hooks script download as well in order to get the script text into the debugger and they are hooked in before me. i'll have to have a look at this.

**Comment by dequis** [comment]
News about this? I just realized i have this extension installed and never use it because I'm not using firebug anymore (Gave up due to its slowness, the built-in tools are decent enough)
